app/models/document.rb

```diff
@@ -7,7 +7,7 @@
 #
 # This model is mutable
 class Document < ApplicationRecord
-  attr_readonly :content_id, :locale, :document_type_id
+  attr_readonly :content_id, :locale
```

Comment:
I think this third commit should be fixed up to the first commit.

Reply:
👍